noxfile.py

```diff
@@ -63,8 +64,9 @@ def version_string_to_tuple(version: str) -> Tuple[int, ...]:
     return tuple(map(int, version.split(".")))
 
 
-@nox.session(python=PYTHON_VERSIONS)  # type: ignore
+@nox.session(python=["3.10"])  # type: ignore
 def lint(session: Session) -> None:
+    # Note: Linting only runs on Python 3.10 to avoid running it on every CI job
```

Comment:
If we only run lints on 3.10, might we run into issues if e.g. we use some new python syntax that's supported in python3.8 but not supported in python3.10? The 3.10 linter would fail to warn us in such a situation.